jenkins/helper/testing_runner.py

```diff
@@ -401,6 +401,8 @@ def generate_crash_report(self):
                 if one_file.exists():
                     try:
                         shutil.move(str(one_file.resolve()), str(core_dir.resolve()))
+                    except shutil.Error as ex:
+                        print(f"failed to move file while cleaning up temporary files {ex}")
```

Comment:
I see no reason why this should happen with the previous cleaning anymore, so it seems sensible to _not_ catch that error now.

Reply:
but if it does, we want the report anyways!